pkg/yurtctl/cmd/revert/revert.go

```diff
@@ -32,6 +32,7 @@ import (
 	"github.com/openyurtio/openyurt/pkg/projectinfo"
 	"github.com/openyurtio/openyurt/pkg/yurtctl/constants"
 	"github.com/openyurtio/openyurt/pkg/yurtctl/lock"
+	enutil "github.com/openyurtio/openyurt/pkg/yurtctl/util/edgenode"
```

Comment:
Better to move it to make it comply with the previous rules.

Reply:
sorry,  which rule do you mean?

the go pkg，third pkg, self pkg. 
following this rule,  `enutil "github.com/openyurtio/openyurt/pkg/yurtctl/util/edgenode"`  shall be placed here 
after `yurtctl/lock`

file `pkg/yurtctl/cmd/convert/convert.go` is not followed, but what about this ?

Reply:
sorry, it's right here.